payments-core/src/main/java/com/stripe/android/Stripe.kt

```diff
@@ -1672,20 +1675,50 @@ class Stripe internal constructor(
      * @param stripeAccountId Optional, the Connect account to associate with this request.
      * By default, will use the Connect account that was used to instantiate the `Stripe` object, if specified.
      * @param callback a [ApiResultCallback] to receive the result or error
+     * @param activity the owning activity. This will be used to identify fraud via a passive (invisible) hCaptcha
      */
     @UiThread
     @JvmOverloads
     fun createRadarSession(
         stripeAccountId: String? = this.stripeAccountId,
-        callback: ApiResultCallback<RadarSession>
+        callback: ApiResultCallback<RadarSession>,
+        activity: AppCompatActivity? = null
     ) {
         executeAsyncForResult(callback) {
             stripeRepository.createRadarSession(
                 ApiRequest.Options(
                     apiKey = publishableKey,
                     stripeAccount = stripeAccountId
                 )
-            )
+            ).flatMap { radarSession ->
+                val siteKey = radarSession.passiveCaptchaSiteKey
+                if (activity == null && BuildConfig.DEBUG) {
```

Comment:
This will be false anytime a merchant consumes our SDK (we publish in release mode).

Reply:
This is specifically intended for our own test apps, and mirrors the existing `StripeCardScanProxy`. In the event I'm trying to track down what went wrong with our test merchant, it'd be nice to have a specific error I can identify within the code.